chibuezeayogu/ecommerce-api
# Implement orders endpoints

**What does the PR do?**
- This PR implements `Orders` endpoints

**Description of Task to be completed?**
- Ensures users can post orders
- Ensures users can view a particular order
- Ensures users can view customers orders
- Ensures users can view short order details
- Ensure test is passing 

**How should this be manually tested?**
- Clone the repository [Ecommerce-API](https://github.com/chibuezeayogu/Ecommerce-API.git) and
-  cd into `Ecommerce-API` and checkout to this branch `ft-implement-orders-endpoints`
- run `bundle install` to install new `gems`
- run `rails server` and test these routes using `Postman`
           ` POST /orders` 
            `GET /orders/orders_id`  
            `GET /orders/inCustomer` 
            `GET /orders/shortDetails` 
- run `rspec` to ensure all tests are passing
